### PR TITLE
Editorial review of Clause 27 [input.output]

### PR DESCRIPTION
**Pull request by tkoeppe**
This is @AlisdairM's work, just repackaged.

### CODE REVIEW

File: source/iostreams.tex [hunk 1]
```diff
@@ -5916,6 +5916,19 @@
 \postcondition
 \tcode{rdbuf() == sb}.
 
+\indexlibrary{\idxcode{basic_ostream}!constructor}%
```
Comment (tkoeppe) [inline source/iostreams.tex]: @AlisdairM: Same question here, how come the synopsis does not list the move constructor?

Comment (AlisdairM) [inline source/iostreams.tex]: Looks like a mis-application of http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2005/n1862.html after the Oxford 2007 meeting that no-one had noticed yet!  I think the declaration in the class definition could be added editorially, given it was present in the original paper.

Comment (AlisdairM) [inline source/iostreams.tex]: Ah, and the other half of the mystery resolved! LWG 911 made these members protected, which is why they show up at the /end/ of the class definition - but the declarations are present and correct.

Comment (tkoeppe) [inline source/iostreams.tex]: Ahhh - thanks! I added reference comments to the synopsis, too.

**Pull request merged**